templates/careers.mustache

```diff
@@ -32,9 +32,9 @@
         <div class="get-started">
           <h2>Get Started Now</h2>
           <p><strong>An Opportunity Center can help you get started on one of these career paths.</strong></p>
-          <p>Start by signing up for the Opportunity Center job system and receive a customized referral to a center.</p>
+          <p>Sign up for the Opportunity Center job system and receive a personalized reccommendation for next steps and a follow-up from a career advisor.</p>
```

Comment:
ahhaha you fixed that before I submitted the review and then I couldn't remove this comment